backend/src/main/java/com/capstone/moneytree/controller/AlpacaController.java

```diff
@@ -0,0 +1,112 @@
+package com.capstone.moneytree.controller;
+
+import com.capstone.moneytree.facade.MarketInteractionsFacade;
+import net.jacobpeterson.domain.alpaca.account.Account;
+import net.jacobpeterson.domain.alpaca.portfoliohistory.PortfolioHistory;
+import net.jacobpeterson.domain.alpaca.position.Position;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import javax.validation.Valid;
+import javax.validation.constraints.NotBlank;
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.Optional;
+
+@RestController
+@RequestMapping("/api/alpaca")
+public class AlpacaController extends ApiController {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(StockController.class);
+    private final MarketInteractionsFacade marketInteractionsFacade;
+
+    @Autowired
+    public AlpacaController(MarketInteractionsFacade marketInteractionsFacade) {
+        this.marketInteractionsFacade = marketInteractionsFacade;
+    }
+
+    /**
+     * Get an Alpaca account endpoint.
+     * @return ResponseEntity of Account.
+     */
+    @GetMapping("/account")
+    public ResponseEntity<Account> getAccount() {
+        Account account = marketInteractionsFacade.getAccount();
```

Comment:
is the getAccount user level or application level? (we get the account of the user?
)

Reply:
Yeah we get all of an Alpaca user. https://alpaca.markets/docs/api-documentation/api-v2/account/

Reply:
But how do we specify the user?

Reply:
Alpaca will work with an OAuth ID later when we configure authentication. We'll get the account of a user with its secret key using the auth secret of the properties. Let's discuss about it more next sprint.